### ISSUE TITLE
Spinner renders text prop twice

### ISSUE THREAD

**Issue by petermakowski** [canonical-web-and-design/react-components#729]
### Actual
Spinner renders text prop twice

**source code**
```
    <i
      className={classNames("p-icon--spinner", "u-animation--spin", {
        "is-light": isLight,
      })}
    >
      {text ? text : "Loading"}
    </i>
    {text && (
      <>
        &ensp;
        <span>{text}</span>
      </>
    )}
```
**Example from MAAS UI**
![image](https://user-images.githubusercontent.com/7452681/158569767-7f5dc6c3-9361-4038-95fd-1ba1f9d7c6cd.png)

### Expected
Supplied text should only be rendered once

### Suggested solution
Text in the icon should only be rendered if no alternative text supplied (and act as a visually hidden description that's accessible for screen readers)

**Comment by bartaz** [comment]
Closed via #762
